README.md

```diff
@@ -35,75 +40,88 @@ For a list of the windows features you can install, please visit this [technet a
 
 * windowsfeature makes use of Powershell so you will need to have at least version 2.0 installed in order to use this module.
 
-## Beginning
+## Usage
 
-To install a single windows feature such as dotnet 3.5:
+To install a single windows feature such as .NET 3.5:
 
-    windowsfeature { 'NET-Framework-Core':
-      ensure => present,
-    }
+```puppet
+windowsfeature { 'NET-Framework-Core':
+  ensure => present,
+}
+```
 
 To install several windows features as part of a large application such IIS:
 
+```puppet
+$iis_features = ['Web-Server','Web-WebServer','Web-Asp-Net45','Web-ISAPI-Ext','Web-ISAPI-Filter','NET-Framework-45-ASPNET','WAS-NET-Environment','Web-Http-Redirect','Web-Filtering','Web-Mgmt-Console','Web-Mgmt-Tools']
 
-    $iis_features = ['Web-Server','Web-WebServer','Web-Asp-Net45','Web-ISAPI-Ext','Web-ISAPI-Filter','NET-Framework-45-ASPNET','WAS-NET-Environment','Web-Http-Redirect','Web-Filtering','Web-Mgmt-Console','Web-Mgmt-Tools']
-
-    windowsfeature { $iis_features:
-      ensure => present,
-    }
+windowsfeature { $iis_features:
+  ensure => present,
+}
+```
 
 To install any associated management tools:
 
-    windowsfeature { 'Web-WebServer':
-      ensure => present,
-      installmanagementtools => true
-    }
+```puppet
+windowsfeature { 'Web-WebServer':
+  ensure                 => present,
+  installmanagementtools => true,
+}
+```
 
-To install all subfeatures:
+To install all subfeatures without having to list them all out:
 
-    windowsfeature { 'Web-WebServer':
-      ensure             => present,
-      installsubfeatures => true
-    }
+```puppet
+windowsfeature { 'Web-WebServer':
+  ensure             => present,
+  installsubfeatures => true,
+}
+```
 
 To install a feature and that requires a restart:
 
-    windowsfeature {'RDS-RD-Server':
-      ensure             => present,
+```puppet
+    windowsfeature { 'RDS-RD-Server':
+      ensure  => present,
```

Comment:
indent here doesn't match the previous lines. I think because it's in a code block, don't need to indent in the block as well.

Reply:
Fixed 👍